parquet-hadoop/src/test/java/org/apache/parquet/format/converter/TestParquetMetadataConverter.java

```diff
@@ -252,4 +267,47 @@ public void randomTestFilterMetaData() {
     }
   }
 
+  @Test
+  public void testNullFieldMetadataDebugLogging() throws NoSuchFieldException, IllegalAccessException, IOException {
```

Comment:
@nezihyigitbasi, thanks for adding a test! I implemented the same fix in #211, but it's much better to have one.